src/components/views/auth/AuthHeader.js

```diff
@@ -27,9 +28,16 @@ export default class AuthHeader extends React.Component {
     render() {
         const AuthHeaderLogo = sdk.getComponent('auth.AuthHeaderLogo');
         const LanguageSelector = sdk.getComponent('views.auth.LanguageSelector');
+        const loginScreen = SdkConfig.get().loginScreen;
+        const showAuthHeaderLogo_Act = loginScreen?.displayActLogo ?? true;
 
         return (
-            <div className="mx_AuthHeader">
+            <div className="mx_AuthHeader"> 
+                {showAuthHeaderLogo_Act &&                  
+                <div>
+                    <br/>              
+                    <img className="mx_AuthHeaderLogo_Act" src="themes/element/img/logos/ACT-Government-inline-back-trans.jpg" alt="ACT Govt Logo"  />                
```

Comment:
As mentioned in another comment, please externalised the img src path and the alt text into the configuration file

Reply:
Done